scripts/testSetup.ts

```diff
@@ -70,8 +70,8 @@ export const getCustomNetworks = async (
   l1Url: string,
   l2Url: string
 ): Promise<{
-  l1Network: L1Network
-  l2Network: Omit<L2Network, 'tokenBridge'>
+  l1Network: ParentChain
+  l2Network: Omit<ChildChain, 'tokenBridge'>
 }> => {
   const l1Provider = new JsonRpcProvider(l1Url)
```

Comment:
Should we rename these inner vars too - eg childProvider, childNetworkInfo etc. I guess the general pattern is just get rid of l1/l2 everywhere. So my comments are the same for this throughout.